digilines/internal.lua

```diff
@@ -20,12 +20,9 @@ function digiline:getAnyInputRules(pos)
 
 	if spec.wire then
 		return digiline:importrules(spec.wire.rules, node)
-	end
-	if spec.effector then
+	elseif spec.effector then
 		return digiline:importrules(spec.effector.rules, node)
```

Comment:
It doesn't actually change the logic.  However, it does turn it into an "else after return", so this change should be reverted